packages/cli/docker/docker-compose.yaml

```diff
@@ -103,5 +104,26 @@ services:
         networks:
             - nango
```

Comment:
unrelated to this PR but we should probably drop support for the ability to run nango from the cli.

Reply:
I'm not even sure what is it for?